src/test/java/io/anserini/collection/JsonDocumentObjectTest.java

```diff
@@ -23,7 +23,12 @@
 import java.util.HashMap;
 import java.util.concurrent.atomic.AtomicInteger;
 
-
+// A file in a JsonCollection can either be:
+// (1) A single JSON object (i.e., a single document)
+// (2) An array of JSON objects
+// (3) JSON Lines (i.e., one JSON object per line)
```

Comment:
IIUC, JsonCollection supports a line/lines of:
(1). JSON object
(2). JSON array

So it supports JSON array lines beside your list, right?

Reply:
> So it supports JSON array lines beside your list, right?

What do you mean by this?

JSON Lines is JSONL: http://jsonlines.org/

Reply:
I see. Then make sense. No need to modify.